Translators/Signs/vsl_tim_generator.py

```diff
@@ -0,0 +1,17 @@
+from itis_codes import ItisCodes
+
+def get_itis_codes():
+    return [ItisCodes.SPEED_LIMIT.value]
+
+def get_geometry(geometry):
+    if len(geometry) == 0:
+        return []
+    annotated_geometry = []
+    # Check if the geometry is a single point
+    if type(geometry[0]) == float:
+        annotated_geometry.append({"latitude": geometry[1], "longitude": geometry[0]})
+        return annotated_geometry

```

Comment:
question: how are we handling a single point VSL in TIM Manager?

Reply:
Currently, we're only able to generate the 'speed limit ahead' TIM with the information in this feed, as we don't have a way to generate the geometry for the section of road that is affected by the variable speed limit. I contacted CDOT to see if there's more detailed information that we could consume, but haven't heard back yet.

Reply:
Is this in accordance with the CTW TIM definitions for VSL? Or will we need to modify?

Reply:
I've double-checked with Rick, and this is in accordance with the CTW TIM definitions for VSL